contracts/core/RebalancingSetToken.sol

```diff
@@ -231,16 +231,17 @@ contract RebalancingSetToken is
             "RebalancingSetToken.propose: Invalid or disabled PriceLibrary address"
         );
 
-        // Assert price divisor is non-zero, ensuring a positive slope
+        // Check that time to pivot is within in 6 hours to 3 days
         require(
-            _auctionPriceDivisor > 0,
-            "RebalancingSetToken.propose: Price divisor must be positive"
+            _auctionTimeToPivot > 21600 && _auctionTimeToPivot < 259200,
```

Comment:
In a following PR, let's allow these params to be passed into the constructor of the RebalancingSetToken and the RebalancingSetTokenFactory

Reply:
Also why are we validating the pivot time in the this file vs. the price curve file?

Reply:
This is valid, the curve should not be aware of the requirements of a proposal in the context of a rebalancing set token.